### PR TITLE
A700

### PR DESCRIPTION
**Pull request by baumanta**
Launch files for multi camera support and running on Intel NUC. 
reduced usage of std::time

### CODE REVIEW

File: local_planner/src/nodes/waypoint_generator.cpp [hunk 1]
```diff
@@ -218,32 +218,38 @@ bool WaypointGenerator::withinGoalRadius() {
...
-    double y_diff = std::abs(goal_.y - pose_.pose.position.y);
-    float goal_acceptance_radius = 1.0f;
-    if (x_diff < goal_acceptance_radius && y_diff < goal_acceptance_radius) {
-    	new_yaw_ = curr_yaw_;
-    }
-  } else {
-    reach_altitude_ = true;
-    getPathMsg();
-  }
+	output_.goto_position.x = planner_info_.offboard_pose.pose.position.x;
```
Comment (bkueng) [inline local_planner/src/nodes/waypoint_generator.cpp]: Indentation: tab -> spaces

File: local_planner/launch/mavros.launch [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<launch>
```
[the remark this answers is not in the document]
Comment (baumanta) [reply local_planner/launch/mavros.launch]: we do not use it for the avoidance. But it is really helpful to test the connetion to the fcu/mavlink router  on A700

File: local_planner/launch/realsense_params.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (mrivi) [reply local_planner/launch/realsense_params.sh]: maybe in `local_planner/resources/`?

File: local_planner/launch/local_planner_A700_1cam.launch [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    <node pkg="tf" type="static_transform_publisher" name="tf_front_camera"
+          args="0.30 0.32 -0.11 0 0 0 fcu front_camera_link 10"/>
+          
+          
+          
+    <arg name="headless" default="false"/>
+    <arg name="ns" default="/"/>
+    <arg name="build" default="posix_sitl_default"/>
+    <arg name="fcu_url" default="/dev/ttyUSB0:921600"/>
+    <!-- <arg name="fcu_url" default="udp://:14540@localhost:14557"/> -->
```
[the remark this answers is not in the document]
Comment (baumanta) [reply local_planner/launch/local_planner_A700_1cam.launch]: I do not want to delete those lines yet, we probably have to switch back to udp again soon

**Pull request merged**